WooCommerce/Classes/ViewRelated/Dashboard/Settings/POS/POSEligibilityChecker.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import UIKit
+import class WooFoundation.CurrencySettings
+import enum WooFoundation.CountryCode
+import protocol Experiments.FeatureFlagService
+import struct Yosemite.SiteSetting
```

Comment:
Do you know if there are any performance effects when importing specific structs, enums, classes?

Reply:
I don't think there's performance gain from this, but this makes it clear what the exact dependencies are if we ever want to move it to a framework.